pom.xml

```diff
@@ -8,8 +8,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>javax.validation</groupId>
-    <artifactId>validation-api</artifactId>
+    <groupId>jakarta.validation</groupId>
+    <artifactId>jakarta.validation-api</artifactId>
```

Comment:
As per @starksm64 I would keep the old artifactId: validation-api. It will be less disruptive that way.

Reply:
We decided to go this way for the one we already published. All the Jakarta artifacts are `jakarta.something`.

See https://search.maven.org/search?q=g:jakarta.validation
and https://search.maven.org/search?q=g:jakarta.*

Reply:
BTW, right now, it uses the artifact we already published.

Reply:
yes I saw that later. I'm making a case here https://bugs.eclipse.org/bugs/show_bug.cgi?id=548104#c31
We will see but let's not block.